oggm/core/flowline.py

```diff
@@ -21,6 +21,7 @@
 from oggm import entity_task
 import oggm.core.massbalance as mbmods
 from oggm.core.centerlines import Centerline, line_order
+from oggm.core.inversion import shape_factor_adhikari, shape_factor_huss
```

Comment:
I referenced the shape factor functions from the inversion module. IMHO, having them in a separate dedicated module and referencing this module from inversion and flowline would make more sense in regards of code structure.

Reply:
Yes, probably. It could be added to ``utils`` for now (and the test moved to ``test_utils`` for now, even if the utils module is definitely overcrowded and could need a split-up one day. What do you think?

Reply:
e.g. move to oggm.utils?